File: workspace/people/p03-security-grc-analyst/home/downloads/inbox/2026-q3/automation/2026-q3-intake-evidence-package-validation-tool.py
# spine_traceability: org.cobalt-harbor=Cobalt Harbor Systems | period.2026-q3=2026-Q3 governance planning period 2026-07-01 through 2026-09-30
"""Synthetic AC-27 intake validator; no network or filesystem operations."""

RECORD = {"artifact_id": "p03-full-008990", "audit_request": "AR-260713-027", "case": "CASE-260713-184", "control": "AC-27", "disposition": "synthetic intake normalized; bounded review retained", "evidence_record": "EV-184-19", "evidence_set": 184, "network_note": "No live endpoints, raw events, or routable addresses are represented.", "organization": "Cobalt Harbor Systems", "program": "Aegis Assurance", "record_id": "INB-164", "review_date": "2026-07-13", "review_period": "2026-Q3", "risk": "RISK-031", "source_id": "p03-src-008990", "unique_marker": "foundation-b001-164"}

def validate(record: dict[str, object]) -> bool:
    required = {"control", "audit_request", "case", "evidence_set", "risk"}
    return required.issubset(record) and record["control"] == "AC-27" and record["evidence_set"] == 184

if __name__ == "__main__":
    assert validate(RECORD), "synthetic intake record is incomplete: foundation-b001-164"

File: workspace/people/p03-security-grc-analyst/home/downloads/inbox/2026-q3/automation/2026-q3-intake-evidence-package-validation-tool.rs
// Review period: 2026 Q3. This intake evidence package validation tool record was assessed using control boundary and operating ownership; the reviewer identified a corroboration need.
pub const REVIEW_PERIOD: &str = "2026 Q3";
pub fn review_summary() -> Vec<(&'static str, &'static str)> {
    vec![
    ("artifact_id", "p03-full-004338"),
    ("organization", "Cobalt Harbor"),
    ("risk_reference", "RISK-031"),
    ("review_date", "2026-07-20"),
    ("evidence_question", "Which source fact would cause the reviewer to accept the control conclusion?"),
    ("evidence_instrument", "control-to-evidence map"),
    ("review_observation", "the evidence sequence exposed a material dependency"),
    ("challenge_performed", "a generic assurance phrase was rejected pending source facts"),
    ("review_profile", "the evidence set was collected before the approval meeting; the evidence custodian confirmed the record boundary; the reviewer sampled a routine operating condition; the source fact supported acceptance without qualification; the comparison used the current control objective; the evidence was retained with the decision record; the exception path was considered and recorded; the reviewer assessed a handover impact; the decision was suitable for routine monitoring; the source chronology was complete; the control claim was independently corroborated; the review boundary excluded unavailable material; the next assurance trigger is completion of the follow-up action; the disposition was communicated to the owner; the retained rationale supports repeat review; the record remains a local offline assurance artifact"),
    ("counterexample", "A repeated control statement does not establish operation when no counterexample was considered."),
    ("decision_rule", "Escalate conflicting observations to the designated reviewer rather than resolving them through filename or identifier changes."),
    ("disposition", "accepted with a retained review note"),
    ("operating_boundary", "No Kio capability is used or required for this offline review.")
    ]
}
